src/database/mysql/mysql-upgrade.xml

```diff
@@ -147,10 +147,8 @@
     </version>
     <version number="24" remark="sort_key">
         <script>ALTER TABLE `mt_cds_object` ADD `sort_key` varchar(GRBMAX) default NULL COLLATE utf8_general_ci</script>
-        <script>
-            UPDATE `mt_cds_object` SET `sort_key`=`dc_title`;
-            ALTER TABLE `mt_cds_object` ADD KEY `mt_cds_object_title` (`dc_title`);
-            ALTER TABLE `mt_cds_object` ADD KEY `mt_cds_object_sort_key` (`sort_key`);
-        </script>
+        <script>UPDATE `mt_cds_object` SET `sort_key`=`dc_title`;</script>
+        <script>ALTER TABLE `mt_cds_object` ADD KEY `mt_cds_object_title` (`dc_title`);</script>
+        <script>ALTER TABLE `mt_cds_object` ADD KEY `mt_cds_object_sort_key` (`sort_key`);</script>
```

Comment:
In this case you do not the semicolon. Please also add the `DROP KEY` statement before the add key to avoid issues with previous runs.

Reply:
The problem is not a key, but the problem is that the first script - creating the `sort_key` column - was executed successfully. Running it again leads to the error that the column already exists ("Duplicate column name 'sort_key'") as I wrote in my bug report.

`drop key` does not help in this case.